src/helpers/amazon-q/amazon-q-client.ts

```diff
@@ -49,17 +52,53 @@ export const callClient = async (
     conversationId: string;
     parentMessageId: string;
   }
-): Promise<ChatSyncCommandOutput> => {
-  const input = {
-    applicationId: env.AMAZON_Q_APP_ID,
-    clientToken: uuid(),
-    userMessage: message,
-    ...(attachments.length > 0 && { attachments }),
-    ...context
-  };
+): Promise<ChatSyncCommandOutput | Error> => {
+  try {
+    message = truncateMessageIfNeeded(message);
```

Comment:
I meant that the truncateMessage and try-catch block should be in 2 new functions in the helper class. The helper class will call the client class' methods

Reply:
Ack. I understand what you mean now -> refactored code to have the bulk of logic be in amazon-q-helpers.ts instead of amazon-q-client.ts